package.json

```diff
@@ -35,30 +35,28 @@
     "test.watch": "yarn test --watch"
   },
   "peerDependencies": {
-    "@stoplight/markdown-viewer": "^4",
-    "@stoplight/ui-kit": "^3",
+    "@stoplight/markdown-viewer": "^5",
     "react": ">=16.8",
     "react-dom": ">=16.8"
   },
   "dependencies": {
     "@fortawesome/free-solid-svg-icons": "^5.15.2",
     "@stoplight/json": "^3.10.0",
     "@stoplight/json-schema-tree": "^2.0.1",
-    "@stoplight/mosaic": "1.0.0-beta.46",
+    "@stoplight/mosaic": "1.0.0-beta.58",
+    "@stoplight/mosaic-code-viewer": "1.0.0-beta.58",
     "@stoplight/react-error-boundary": "^1.0.0",
     "@types/json-schema": "^7.0.7",
     "classnames": "^2.2.6",
     "lodash": "^4.17.19"
   },
   "devDependencies": {
-    "@rollup/plugin-typescript": "^3.1.1",
     "@sambego/storybook-state": "^1.3.6",
     "@stoplight/eslint-config": "^1.2.0",
-    "@stoplight/markdown-viewer": "^4.3.4",
-    "@stoplight/scripts": "^8.2.0",
+    "@stoplight/markdown-viewer": "^5.0.0-beta.4",
+    "@stoplight/scripts": "9.0.1",
```

Comment:
Are you sure the package can be loaded in node.js-compliant env?
Could we add https://github.com/stoplightio/markdown/blob/beta/package.json#L40?

Reply:
I've added it and looks it's breaking for lodash:
```
SyntaxError: Named export 'last' not found. The requested module 'lodash' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export
```
Though I can see it's also used in markdown so I'll see how it's done there

Reply:
Changing lodash imports to default ones: `import last from 'lodash/{function}.js';`, same way it's done in `markdown` made it work. I just had to add `allowSyntheticDefaultImports`. Would that be enough?